src/test/java/com/google/api/codegen/testdata/discoveries/go/go_compute.v1.json.baseline

```diff
@@ -3555,7 +3559,8 @@ func main() {
   httpHealthCheck := ""  // TODO: Update placeholder value.
 
   rb := &compute.HttpHealthCheck{
-    // TODO: Add desired fields of the request body.
+    // TODO: Add desired fields of the request body to be changed. All existing
```

Comment:
"to be changed" reads confusingly here to me, since it implies only the fields to be changed should be filled in, but the next sentence implies fields not filled in will be overwritten. How can we clarify?